### ISSUE TITLE
Laravel Nova v4.12.14 breaks this field

### ISSUE THREAD

**Issue by Sotiris-k** [wdelfuego/nova-datetime#2]
I just updated my nova installation to the latest version as described in the title and the datetime fields unfortunately no longer display anything.

As mentioned [here](https://github.com/laravel/nova-issues/issues/4665) they messed up the displayUsing function.

You probably don't have to do anything because it's their problem but I'll keep you posted if it's fixed or not in the next version.

**Comment by wdelfuego** [comment]
Thanks for the heads-up, @Sotiris-k !

**Comment by wdelfuego** [comment]
Nova fix is [pending release](https://github.com/laravel/nova-issues/issues/4670#issuecomment-1207391309)

**Comment by Sotiris-k** [comment]
It's fixed in Nova v4.13.0

**Comment by Jon78** [comment]
Partly fixed:
```
DateTime::make('Publicatie', 'published_at')->displayUsing(fn ($value) => $value ? $value->diffForHumans() : null)
```
shows
![Schermafdruk van 2022-08-26 10-22-51](https://user-images.githubusercontent.com/51792370/186857719-813ce422-4850-4cd9-982e-9c358a832b58.png)
